hudi-console/hudi-console/src/main/assembly/script/schema/pgsql-schema.sql

```diff
@@ -0,0 +1,812 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- ----------------------------
+-- drop table if exists
+-- ----------------------------
+
+drop table if exists "public"."t_user";
+drop table if exists "public"."t_permissions";
+drop table if exists "public"."t_setting";
+drop table if exists "public"."t_role";
+drop table if exists "public"."t_role_menu";
+drop table if exists "public"."t_menu";
+drop table if exists "public"."t_message";
+drop table if exists "public"."t_flink_tutorial";
+drop table if exists "public"."t_flink_sql";
+drop table if exists "public"."t_flink_savepoint";
+drop table if exists "public"."t_flink_project";
+drop table if exists "public"."t_flink_env";
+drop table if exists "public"."t_flink_effective";
+drop table if exists "public"."t_flink_config";
+drop table if exists "public"."t_flink_cluster";
+drop table if exists "public"."t_flink_app";
+drop table if exists "public"."t_app_build_pipe";
+drop table if exists "public"."t_app_backup";
+drop table if exists "public"."t_alert_config";
+drop table if exists "public"."t_access_token";
+drop table if exists "public"."t_flink_log";
+drop table if exists "public"."t_team";
+drop table if exists "public"."t_variable";
+drop table if exists "public"."t_external_link";
+drop table if exists "public"."t_yarn_queue";
+
+-- ----------------------------
+-- drop sequence if exists
+-- ----------------------------
+drop sequence if exists "public"."streampark_t_user_id_seq";
```

Comment:
`streampark_t_user_id_seq` -> `hudi_t_user_id_seq`?

Do the same below. `streampark` -> `hudi`